.teamcity/src/main/kotlin/configurations/Gradleception.kt

```diff
@@ -130,7 +130,7 @@ class Gradleception(
 
             localGradle {
                 name = "QUICKCHECK_WITH_GRADLE_BUILT_BY_GRADLE"
-                tasks = "clean sanityCheck test -PflakyTests=exclude " + if (bundleGroovy4) "--dry-run" else ""
```

Comment:
Does this mean we're able to run the full Gradleception build?  Do we still have a memory leak problem?

Reply:
Yes, I'm not sure if there's a memory leak because it has rare reproducibility -- I was going to leave this on and see if we need to disable again later. I could try to run it locally many times but IIRC I never reproduced it.